### DISCUSSION TITLE
Daily 24/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#447] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by rpgrca** [comment]
Ayer hice solucioné varios arreglos que había reportado, también tuve la clase de web donde no vimos mucho, alguien pidió ver servicios así que explicó otra vez lo que ya tendrían que haber hecho. La clase que viene va a ser lo de Django pero ya va a ser tarde para nosotros así que hoy en cuanto pueda me pongo a repasar lo del view y armar un video para explicar cómo hacerlo para que cada uno pueda hacer al menos uno.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve terminando algunas tareas y las subí al repositorio. Tuve un día complicado, así que no pude hacer más que eso.
Hoy trataré de terminar con todas las tareas asignadas. Por la noche tengo clase de Inglés y también tengo tareas para realizar de esa materia. Esta semana no tendré clase de Programación, así que esperaré el video que hará Roberto para tratar de entender las tareas para la parte de backend, ya que no hay material subido a la plataforma y seguramente no lo habrá hasta la semana que viene.

**Comment by BraianTroncoso** [comment]
Ayer solucioné un par debugs como habia dicho. Estuve complicado porque tenia otros trabajos practicos pero pude realizar todo con éxito.
